website/content/en/docs/concepts/peers.md

```diff
@@ -76,7 +76,7 @@ Before proceeding, ensure that the CA certificate (the CA private key is not nee
 Install the ClusterLink operator by running the following command:
 
 ```sh
-clusterlink deploy peer --autostart --name <peer_name> --fabric <fabric_name>
+clusterlink deploy peer --name <peer_name> --fabric <fabric_name>
```

Comment:
should you add a note that this starts all and the user can also set `--start` explicitly?

Reply:
I specify in more detail in the deployment operator section.